lms/djangoapps/completion/models.py

```diff
@@ -217,3 +217,19 @@ def __unicode__(self):
             block_key=self.block_key,
             completion=self.completion,
         )
+
+
+class UserSalesforceContactId(models.Model):
+    """
+    Model to track the Salesforce Contact ID 18 user value.
+
+    The field contact id is unique for each user.
+
+    The user contact id was initially stored in the model UserSocialAuth extra_data.
+
+    A new Django command is available to grab the user contact id from UserSocialAuth
+    and then create the new record in this model.
+    """
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
could you add `db_index=True` in order to improve the searches

Reply:
Got it, thank you. @andrey-canon